charts/spire/charts/spiffe-oidc-discovery-provider/templates/podmonitor.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.insecureScheme.enabled }}
```

Comment:
Why only if there is an insecure Scheme? or is this a copy paste thing you missed.

Reply:
That prometheus export only exists if the insecure Scheme flag is set. its the nginx monitor.